README.md

```diff
@@ -95,6 +95,6 @@ SmartBugs provides a collection of vulnerable Solidity smart contracts organized
 
 
 ## Contributing to SmartBugs
-Everyone is welcome to contribute to the development of SmartBugs. 
+♺ We welcome contribute to the development of SmartBugs. 
 
-The easiest way is to create a pull request. Please write your commit messages in the present tense. We suggest you follow this guide: [Commit Message Guidelines](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53).
+The easiest way is to create a pull request. We suggest you follow this guide: [Commit Message Guidelines](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53).
```

Comment:
I like the use of present tense. Perhaps we should suggest the "imperative mood" (as in [https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=HEAD#n133](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=HEAD#n133).

Perhaps we could remove the reference to the "Commit Message Guidelines" and just include something like "In your commit messages, please describe your changes in imperative mood. E.g. "Add support for tool XYZ" instead of "I added support for tool XYZ"."

Reply:
sure.